src/components/AppointmentList/AppointmentList.component.jsx

```diff
@@ -1,29 +1,86 @@
 import React from "react";
 import { Avatar, Button, List, Typography } from "antd";
 import { CalendarOutlined } from "@ant-design/icons";
+import { countDayBetween } from "../../utils/convert.utils";
+import moment from "moment";
+import { convertMomentToMilliseconds } from "../../utils/convert.utils";
 
 const AppointmentListComponent = ({ appointmentListData }) => {
   return (
-    <List
-      itemLayout="horizontal"
-      dataSource={appointmentListData ? appointmentListData : []}
-      renderItem={(appointment) => (
-        <List.Item>
-          <List.Item.Meta
-            avatar={<Avatar icon={<CalendarOutlined />} size={48} />}
-            title={
-              <Typography.Title
-                level={5}
-              >{`Appointment Id: ${appointment?.id}`}</Typography.Title>
-            }
-            description={`Clinic: ${appointment?.clinic?.name}`}
-          />
-          <Button type="link" onClick={() => appointment?.onClick()}>
-            Xem chi tiết
-          </Button>
-        </List.Item>
-      )}
-    />
+    <>
+      <List
+        className="appointment-list-pc"
+        itemLayout="horizontal"
+        dataSource={appointmentListData ? appointmentListData : []}
+        renderItem={(appointment) => (
+          <List.Item>
+            <List.Item.Meta
+              avatar={<Avatar icon={<CalendarOutlined />} size={48} />}
+              title={
+                <Typography.Title level={5}>
```

Comment:
shoulde the warning text be in the different line instead of being in the same line of id like this ?

Reply:
yeah becuz there is not much space in mobile,